### PR TITLE
Remove repeated words

### PR DESCRIPTION
**Pull request by ElyesH**

### CODE REVIEW

**Comment by igalic** [comment]
somehow didn't exist Typos in license text…

File: usr.sbin/ppp/physical.h [hunk 1]
```diff
@@ -9,7 +9,7 @@
  * conditions, because large sections american law seems to have been
  * created by a bunch of jerks on drugs that are now illegal, forcing
  * me to include this copyright-stuff instead of placing this in the
- * public domain.  The name of of 'Yes Interactive' or 'Eivind Eklund'
+ * public domain.  The name of 'Yes Interactive' or 'Eivind Eklund'
```
Comment (bsdimp) [inline usr.sbin/ppp/physical.h:12]: We need Yes Interacive and Eivind Eklund's permission for these.

File: usr.sbin/sesutil/eltsub.c [hunk 2]
```diff
@@ -12,7 +12,7 @@
  *    derived from this software without specific prior written permission.
  *
  * Alternatively, this software may be distributed under the terms of the
- * the GNU Public License ("GPL").
+ * GNU Public License ("GPL").
```
Comment (bsdimp) [inline usr.sbin/sesutil/eltsub.c:15]: We'd need Matthew Jacob's permission for these...  And I'm not sure how that would go.

**Comment by ElyesH** [comment]
ping

**Pull request closed without merging**